src/main/java/com/faforever/client/query/SpecificationController.java

```diff
@@ -340,6 +342,9 @@ private Condition getStringCondition(ComparisonOperator comparisonOperator, Stri
     if (comparisonOperator == RE) {
       return prop.eq("*" + value + "*");
     }
+    if (comparisonOperator == EX) {
+      return prop.ne("*" + value + "*");
```

Comment:
Did you test it?

Reply:
yes. it works correctly. All tests is passed